static/js/unacked.js

```diff
@@ -0,0 +1,32 @@
+import * as Table from './table.js'
+import { UrlDataSource } from './datasource.js'
+
+const search = new URLSearchParams(window.location.hash.substring(1))
+const queue = search.get('name')
+const vhost = search.get('vhost')
+const url = `api/queues/${encodeURIComponent(vhost)}/${encodeURIComponent(queue)}/unacked`
+const channelBaseUrl = 'channel#name='
+document.getElementById('queue-link').href = `/queue#vhost=${encodeURIComponent(vhost)}&name=${encodeURIComponent(queue)}`
+document.getElementById('queue-link').innerHTML = queue
+document.querySelector('#pagename-label').textContent = queue + ' in virtual host ' + vhost
+
+const tableOptions = {
+  dataSource: new UrlDataSource(url, { useQueryState: false }),
+  keyColumns: ['message_tag', 'consumer_tag', 'channel_name', 'unacked_for_seconds'],
```

Comment:
Not sure, but `message_tag` and `channel_name` should be enough as key?

```suggestion
  keyColumns: ['message_tag', 'channel_name'],
```

Reply:
Ah! That's because this is done:
https://github.com/cloudamqp/lavinmq/blob/b4b9045f7af679714308bd1c1fb673e1ee412c92/static/js/unacked.js#L23-L33

Try this instead:
```javascript
Table.renderTable('table', tableOptions, function (tr, item, firstRender) {
  if (firstRender) {
    Table.renderCell(tr, 0, item.message_tag)
    Table.renderCell(tr, 1, item.consumer_tag)
    const channel = document.createElement('a')
    channel.href = channelBaseUrl + encodeURIComponent(item.channel_name)
    channel.textContent = item.channel_name
    Table.renderCell(tr, 2, channel)
  }
  Table.renderCell(tr, 3, item.unacked_for_seconds, 'right')
  Table.renderCell(tr, 4, item.prefetch_count, 'right')
})
```

Reply:
Aha! 👍